pkg/cmd/plugincheck/main.go

```diff
@@ -30,75 +20,34 @@ Y88b 888Y8b.    888 d88P888    Y8b.    Y88b.   888  888Y88b. Y8b.    Y88b 888
                 888
                 888
 
-	**plugincheck V1 is deprecated and you should not use it.
+	**plugincheck V1 is no longer supported.**
 
 	Use V2 instead: plugincheck2
 
-	To install it see README https://github.com/grafana/plugin-validator`)
-	fmt.Println()
-	var (
-		strictFlag  = flag.Bool("strict", false, "If set, plugincheck returns non-zero exit code for warnings")
-		privateFlag = flag.Bool("private", false, "If set, plugincheck reports private signature check error as warning")
-	)
-
-	flag.Parse()
-
-	if len(flag.Args()) < 1 {
-		fmt.Fprintln(os.Stderr, "missing plugin url")
-		os.Exit(1)
-	}
-
-	pluginURL := flag.Arg(0)
-
-	schemaFile, err := ioutil.TempFile("", "plugin_*.schema.json")
-	if err != nil {
-		fmt.Fprintln(os.Stderr, "couldn't create schema file")
-		os.Exit(1)
-	}
-	defer os.Remove(schemaFile.Name())
+	To install it see README https://github.com/grafana/plugin-validator\n`)
```

Comment:
This will print a literal "\n" in the output, because escape sequences aren't parsed when using strings quoted by backticks

```suggestion
	To install it see README https://github.com/grafana/plugin-validator` + "\n")
```
Alternatively, you can use:
```suggestion
	To install it see README https://github.com/grafana/plugin-validator
`)
```

Because new lines will be printed literally, however I personally find the first suggestion easier to understand :)